### PR TITLE
Filter list of control-plane app metrics

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Restrict coredns and etcd  metrics

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -998,18 +998,53 @@ prometheus-operator:
...
+        # coredns_forward_request_count_total
+        # process_cpu_seconds_total
+        # process_open_fds
+        # process_resident_memory_bytes
+        - url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.control-plane.coredns
           writeRelabelConfigs:
             - action: keep
               regex: coredns
               sourceLabels: [job]
+            - action: keep
```
Comment (frankreno) [inline deploy/helm/sumologic/values.yaml:1018]: Just curious on the reason to have 2 keep blocks vs something like this:

```
        - url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.control-plane.coredns
           writeRelabelConfigs:
             - action: keep
               regex: coredns;coredns_cache_(size|(hits|misses)_total)|coredns_dns_request_duration_seconds_(count|sum)|coredns_(dns_request|dns_response_rcode|forward_request)_count_total|process_(cpu_seconds_total|open_fds|resident_memory_bytes)
               sourceLabels: [job,__name__]
```

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:1018]: It's more clear in my personal opinion :)
I also wasn't sure how exactly works the writeRelabelConfig for such construct, but I read the docs and I don't see any risk related to that.
I can change it to be more compressed and to keep convention

Comment (frankreno) [reply deploy/helm/sumologic/values.yaml]: The only reason I brought it up was just consistency like we do the other rules. I have no problem with doing it this way either.

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml]: Convention is important :D
I updated PR, need to re-test it :)

**Pull request merged**